schema/m3-room.schema.json

```diff
@@ -0,0 +1,1985 @@
+            "title": "Viewable Item Nodes",
+            "description": "An array of objects describing what item nodes can be viewed from this node, and how.",
+            "items": {
+              "$id": "#/properties/nodes/items/properties/viewableNodes/items",
+              "type": "object",
+              "title": "Viewable Node Object",
+              "description": "Describes what item nodes can be viewed from this node, and how.",
+              "required": [ "id", "strats" ],
+              "additionalProperties": false,
+              "properties": {
+                "id": {
+                  "$id": "#/properties/nodes/items/properties/viewableNodes/items/properties/id",
+                  "type": "integer",
+                  "title": "The ID of the item node that is viewable, within the room.",
+                  "default": 0,
+                  "examples": [
+                    1
+                  ]
+                },
+                "strats" : {
+                  "$id": "#/properties/nodes/items/properties/viewableNodes/items/properties/strats",
+                  "type": "array",
+                  "title": "Viewable Node Strats",
+                  "description": "An array of strats that can be executed to view the item.",
+                  "items": {
+                    "$ref" : "#/definitions/strat",
+                    "$id": "#/properties/nodes/items/properties/viewableNodes/items/properties/strats/items"
+                  }
+                }
+              }
+            }
+          },
+          "yields": {
+            "$ref" : "#/definitions/yields",
+            "$id": "#/properties/nodes/items/properties/yields",
+            "description": "A list of flags that are activated by interacting with this node."
+          },
+          "note": {
+            "$ref" : "m3-note.schema.json#/definitions/note",
+            "$id": "#/properties/nodes/items/properties/note"
+          },
+          "devNote": {
+            "$ref" : "m3-note.schema.json#/definitions/devNote",
+            "$id": "#/properties/nodes/items/properties/devNote"
+          }
+        }
+      }
+    },
+    "links": {
+      "$id": "#/properties/links",
+      "type": "array",
+      "title": "Links; pathways within the Room",
+      "items": {
+        "$id": "#/properties/links/items",
+        "type": "object",
+        "title": "Link",
+        "required": [
+          "from",
+          "to"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "from": {
+            "$id": "#/properties/links/items/properties/from",
+            "type": "integer",
+            "title": "This Link originates at this Node ID within the Room",
+            "default": 0,
+            "examples": [
+              1
+            ]
+          },
+          "to": {
+            "$id": "#/properties/links/items/properties/to",
+            "type": [
+              "array",
+              "null"
```

Comment:
```suggestion
              "array"
```

Reply:
An empty array looks better and doesn't break tests.

And probably also means a clean links section would have a link entry for every node as a `from` value.  And a `to` array to be filled in with the in-room node connections..

Reply:
Agree, the "null" option doesn't make sense. I can remove it in a separate PR after this is merged & executed, if that sounds ok?